admin_guide/scheduling/scheduler-advanced.adoc

```diff
@@ -0,0 +1,53 @@
+[[admin-guide-scheduling-advanced]]
+= Advanced Scheduling
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Advanced scheduling involves configuring a pod so that the pod is required to run on particular nodes or has a preference to run on particular nodes.
+
+Generally, advanced scheduling  is not necessary, as the {product-title} automatically places pods in a reasonable manner. For example, the default scheduler attempts to distribute pods across the nodes evenly and considers the available resources in a node. However, you might want more control over where a pod is placed.
+
+If a pod needs to be on a machine with a faster disk speed (or prevented from being placed on that machine) or pods from two different services need to be located so they can communicate, you can use advanced scheduling to make that happen.
+
+== Using Advanced Scheduling
+
+There are several ways to invoke advanced scheduling in your cluster:
+
+[[admin-guide-scheduling-advanced-pod]]
+Pod Affinity and Anti-Affinity:: Pod affinity allows a *pod* to specify an affinity (or anti-affinity) towards a group of *pods* (for an application's latency requirements, due to security, and so forth) it can be placed with. The node does not have control over the placement.
++
+Pod affinity uses labels on nodes and label selectors on pods to create rules for pod placement. Rules can be mandatory (required) or best-effort (preferred).
++
+See xref:../../admin_guide/advanced_scheduling/pod_affinity.adoc#admin-guide-sched-pod-affinity[Using Pod Affinity and Anti-Affinity].
+
+[[admin-guide-scheduling-advanced-node]]
+Node Affinity:: Node affinity allows a *pod* to specify an affinity (or anti-affinity) towards a group of *nodes* (due to their special hardware, location, requirements for high availability, and so forth) it can be placed on. The node does not have control over the placement.
++
+Node affinity uses labels on nodes and label selectors on pods to create rules for pod placement. Rules can be mandatory (required) or best-effort (preferred).
++
+See xref:../../admin_guide/advanced_scheduling/node_affinity.adoc#admin-guide-sched-affinity[Using Node Affinity].
+
+[[admin-guide-scheduling-advanced-selector]]
+Node Selectors:: Node selectors are the simplest form of advanced scheduling. Like node affinity, it also uses labels on nodes and label selectors on pods to allows a *pod* to control the *nodes* on which is can be placed. However, node selectors do not have required and preferred rules that node affinities have.
```

Comment:
Not sure about "Like node affinity, it also uses labels on nodes and label selectors on pods to allows a *pod* to control the *nodes* on which is can be placed." The first "it" is vague and I think breaks the tense. Maybe:

"Like node affinity, node selectors use labels on nodes and label selectors on pods to allow a *pod* to control the *nodes* on which it can be placed."

Note the s/is/it as well.